frontend/src/components/Layout/MobileMenu/styled.ts

```diff
@@ -0,0 +1,157 @@
+import styled from 'styled-components';
+import { ReactFragment } from 'react';
+import { Link } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core';
+import { ListItemButton, Backdrop } from '@mui/material';
+
+import theme from 'themeSettings';
+
+import { HEADER_HEIGHT_MOBILE } from '../Header/styled';
+
+interface TabWrapperTypes {
+  component?: ReactFragment;
+  to?: string;
+}
+
+export const MobileMenuSlide = styled('div')({
+  width: '176px',
+  height: `calc(100% - ${HEADER_HEIGHT_MOBILE})`,
+  backgroundColor: theme.palette.secondary.main,
+  padding: '8px 16px 40px 8px',
+  position: 'absolute',
+  top: HEADER_HEIGHT_MOBILE,
+  right: '0px',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'flex-start',
+});
+
```

Comment:
We have problem with this item, i am not sure that its concern for this component, but we have little problem on 375px
[https://ibb.co/4RrRwG7](url) - Lets see

Reply:
https://prnt.sc/26qtilp